example.json

```diff
@@ -5,10 +5,19 @@
 	"chassis" : {
 		"type": 10,
 		"pn" : "CHAS-C00L-12",
-		"serial": "45678",
+		/* Fields may be given explicit types by setting to object with keys `type` and `val`*/
+		/* Supported types are: "bcdplus", "6bitascii" and "text" */
+		"serial": {
+		    "type": "text",
+		    "val": "45678"
```

Comment:
Now it's `type` and `data` for `custom`, but `type` and `val` for regular fields.
Make it please `data` here too.

Reply:
Changed `val` to `data`